assets/src/blocks/Gallery/GalleryFrontend.js

```diff
@@ -13,6 +23,20 @@ export const GalleryFrontend = ({
   const layout = getGalleryLayout(className, gallery_block_style);
   const postType = document.body.getAttribute('data-post-type');
 
+  const [lightboxIsOpen, setLightboxIsOpen] = useState(false);
+  const [lightboxIndex, setLightboxIndex] = useState(0);
+
+  const onImageClick = lightboxIndex => {
```

Comment:
I guess this should be named after what the function does, rather than `onImageClick`. If for example there is another way we could open the lightbox at the image, it could also use the exact same function. So maybe something like `openLightBox = (index) =>` would be more useful.

Especially if you see it in JSX:
```
      {layout === 'slider' && <GalleryCarousel onImageClick={openLightbox} images={images || []} />}
```